src/components/Loader/Loader.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withStyles from '@material-ui/core/styles/withStyles';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+const styles = () => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+
+  secondary: {
+    color: '#000000',
+  },
+});
+
+const Loader = (props) => {
+  const {
+    classes,
+    color,
+    customSize,
+    size,
+  } = props;
+
+  const standardSizes = {
+    small: 30,
+    medium: 40,
+    large: 50,
+  };
+
+  return (
+    <div className={classes.root}>
+      <CircularProgress
+        color={classes[color]}
+        size={customSize || standardSizes[size]}
```

Comment:
El prop `size` no será _undefined_ para cualquier caso ya que tiene un valor por defecto, así que podemos validar si el valor enviado es alguno de los tamaños stándar o un valor personalizado (ejemplo 80, '3rem').
```suggestion
        size={['small', 'medium', 'large'].includes(size) ? standardSizes[size] : size}
```